Makefile

```diff
@@ -14,6 +14,10 @@ bindings/BlockInspector/BlockInspector.go:
 	mkdir -p bindings/BlockInspector
 	seer evm generate --package BlockInspector --output bindings/BlockInspector/BlockInspector.go --foundry out/BlockInspector.sol/BlockInspector.json --cli --struct BlockInspector
 
+bindings/Tests.go:
```

Comment:
This should be the path to the output file. `make` will check if the file at this path exists and only run this target if it doesn't.

If you want it to be a target which always runs, you need to list it under .PHONY at the top of the file.

Here's some documentation on `make`: https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html

Reply:
I renamed the path and remove test from bin